pkg/display/column.go

```diff
@@ -84,3 +84,11 @@ func ParseJSONOutputColumnValue(val interface{}, col *queryresult.ColumnDef) (in
 		return ColumnValueAsString(val, col)
 	}
 }
+
+func ParseCSVOutputColumnValue(val interface{}, col *queryresult.ColumnDef) (string, error) {
```

Comment:
Is it necessary to have a separate function for this?

Maybe we could pass in an option to `ColumnValueAsString` and achieve the same thing?